s2/loop_test.go

```diff
@@ -1332,6 +1357,18 @@ func TestLoopRelations(t *testing.T) {
 			contains:   true,
 			sharedEdge: true,
 		},
+		// https://github.com/golang/geo/issues/77 and 78
+		// These cases failed prior to this fix.
```

Comment:
Remove this comment ("this fix" means the pull request, not the file).  I'd suggest a different comment. Something like:
// Cases below here prevent regressions of bugs which previously appeared in the golang version
// and  which are missing test coverage in the C++ codebase.

Reply:
Updated.